feature/migration/qrcode/qr-code-format.md

```diff
@@ -0,0 +1,463 @@
+
+### `SequenceEnd`
+
+- Available since: version 1
+- Type: Integer
+
+Information about multiple accounts might not fit into a single QR code. This property contains the number of QR codes
+used for a single export operation. A reader can use this information together with `SequenceNumber` to figure out how
+many QR codes in a sequence have already been read and how many are still missing.
+
+### `IncomingServer`
+
+- Available since: version 1
+- Type: Array
+
+This type contains properties that are only present once in an account, mostly information about the incoming server .
+The first element in this array is `IncomingProtocol`. Its value determines the contents of the rest of the array.
+
+For forward-compatibility a reader must skip reading an account when an unsupported `IncomingProtocol` value is
+encountered.
+
+Note: Since the size of this array depends on the value of `IncomingProtocol`, future specifications can only add
+properties on a per protocol basis.
+
+### `IncomingProtocol`
+
+- Available since: version 1
+- Type: Integer
+- Values:
+  - 0 (`IMAP`; available since: version 1)
+  - 1 (`POP3`; available since: version 1)
+
+For the values 0 and 1 the contents of the `IncomingServer` array are as follows:
+
+- `IncomingProtocol`
+- `Hostname`
+- `Port`
+- `ConnectionSecurity`
+- `AuthenticationType`
+- `Username`
+- `AccountName` (optional)
+- `Password` (optional)
+
+For forward-compatibility a reader must skip reading an account when an unsupported `IncomingProtocol` value is
+encountered. It must also skip the account if any of the other properties contain unsupported values.
+
+A writer may omit optional elements from the array if the subsequent elements are also omitted.
+
+#### Writing and reading `AccountName`
+
+If the account name is equal to the email address of the first identity, the writer may omit the value. If the element
+can't be omitted (because one of the following elements is present), `null` or the empty string may be used instead.
+
+A reader must use the email address of the first identity as account name if `AccountName` is omitted or its value is
+`null` or the empty string.
+
+#### Writing and reading `Password`
+
+If the writer doesn't want to include the password, it may omit the `Password` element.
+
+For forward-compatibility a reader must treat a `Password` value of `null` or the empty string like an omitted
+password.
+
+### `Hostname`
+
+- Available since: version 1
+- Type: String
+
+A server hostname. Currently only ASCII-only hostnames are allowed. This includes the ASCII Compatible Encoding (ACE) of
+Internationalized Domain Names (IDN).
+
+### `Port`
+
+- Available since: version 1
+- Type: Integer
+- Values: 1-65535
+
+The TCP port used by an incoming or outgoing server.
+
+### `ConnectionSecurity`
+
+- Available since: version 1
+- Type: Integer
+- Values:
+  - 0 (`Plain`; available since: version 1)
+  - 1 (`TryStartTls`; available since: version 1)
```

Comment:
@freaktechnik noted that `TryStartTls` is no longer a valid value in Thunderbird desktop and that it will never be present in a QR code.

I'll update the text to add a placeholder. That way we make sure the value `1` is a) invalid and b) won't be (re)used in future versions of this specification.